scout/server/blueprints/institutes/templates/overview/causatives.html

```diff
@@ -18,13 +18,48 @@
     <a class="nav-link" href="{{ url_for('cases.index') }}">Institutes</a>
   </li>
   <li class="nav-item">
-    <a class="nav-link" href="{{ url_for('overview.cases', institute_id=institute._id) }}">{{ institute.display_name }} cases </a>
+    <a class="nav-link" href="{{ url_for('overview.cases', institute_id=institute._id) }}">{{ institute.display_name }}</a>
   </li>
   <li class="nav-item active">
     <span class="navbar-text">Causative variants</span>
   </li>
 {% endblock %}
 
+{% block content_main %}
```

Comment:
Sorry for the edits! I moved the macro to the end to make the block structure more evident, without changing the content.

Reply:
It's fine, just let us know when you are done if you need a review!

Reply:
Anytime!